### PR TITLE
Setup Hello Microverse

### PR DESCRIPTION
**Pull request by lily4178993**
**### Microverse module 1: Week 1 - First Project**

In this branch, I have:

-  added a HTML and CSS files
-  added a gitignore file
-  added a README file
-  Set up linters for HTML and CSS

### CODE REVIEW

File: LICENCE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Nelly Telli
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
```
Comment (taldr27) [inline LICENCE.md]: - [ ] I noticed that you have multiple `LICENSE` files in your repository, which can cause confusion for contributors. To ensure that your project remains organized and easy to navigate, it's best to remove the duplicate `LICENSE` file.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+- 📖 [About the Project](#about_project)
+  - [Screenshot](#screenshot)
+  - 🛠 [Built With](#built_with)
+- 💻 [Installation](#installation)
+- 👥[Author](#author)
+- 🤝 [Contributing](#contributing)
+- ⭐️ [Show your support](#support)
+- 🙏 [Acknowledgements](#acknowledgements)
+- 📝 [License](#license)
```
Comment (taldr27) [inline readme.md]: - [ ] Excellent job utilizing the Microverse template for your README and personalizing it to represent your project! However, I noticed that some sections from the content table are missing. To make your README file even more comprehensive and meet the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), I kindly recommend adding all the required sections outlined in the [Microverse template](https://github.com/microverseinc/readme-template). Please note that the FAQ section is optional. Keep up the good work! 🎯
Your table of content should contain:
```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)
```

**Review by taldr27 — CHANGES_REQUESTED** [review]
## Hello @lily4178993 , 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Correct use of GitHub Flow.
✅ Linters tests passing.
✅ Nice README file.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: LICENCE.md (hunk 1 above)
Comment (Efiamotu-1) [reply LICENCE.md]: @lily4178993 please you have not attended to this requirement by your previous reviewer, make sure to remove duplicate files as it is unprofessional to have it the way it currently is in your branch :+1:

**Review by Efiamotu-1 — APPROVED** [review]
Hi @lily4178993 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow or Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @taldr27 and @Efiamotu-1 for your advices.

**Pull request merged**